helix-view/src/editor.rs

```diff
@@ -305,6 +305,14 @@ pub struct Editor {
     pub last_completion: Option<CompleteAction>,
 
     pub exit_code: i32,
+
+    pub config_events: SelectAll<tokio_stream::Once<ConfigEvent>>,
```

Comment:
Let's instead use `tokio::sync::mpsc::unbounded_channel`, no need for a `SelectAll`